storage/mysql/schema/.skeema

```diff
@@ -0,0 +1,9 @@
+default-character-set=utf8mb4
```

Comment:
It might be irritating for example if people have to add a prod section here or edit the defaults to match their setup. Would they have their own config file in that case?

Reply:
This should work as-is for dev purposes but yes, they'd have to add a section if they want to use it to push updates to their prod database (either by editing this file or using `skeema add-environment`). I'm thinking that I'd automate this for Kubernetes deployments.

Reply:
The alternative would be using a library that executes database migrations from within the server/signer processes, so it already knows the database connection details (e.g. https://github.com/golang-migrate/migrate).

Reply:
If you had e.g. a different server port do you have to keep this file modified locally? Not sure if there is a way to manage this.

Reply:
Mixing dev / prod configs in the same file could also be a recipe for disaster, it was just an example.

Reply:
Unfortunately yes - Skeema doesn't let you override the host or post on the command-line. I don't think mixing dev and prod configs in one file would be too problematic, since you have to specify the config name whenever you invoke a command, so it's fairly explicit about which config you're using.

Reply:
Oh actually, you can have multiple .skeema files and they'll get combined, but they have to be in subdirectories. So it'd be possible to add subdirectories for each environment and .gitignore them.